### PR TITLE
dapp dev quick start NDX

### PR DESCRIPTION
**Pull request by cpengilly**

### CODE REVIEW

File: pages/builders/dapp-developers/quick-start.mdx [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+## Step 1: Get Testnet ETH from Superchain Faucet
+
+You'll need some testnet ETH on OP Sepolia to pay for the [gas fees](https://ethereum.org/en/developers/docs/gas/) associated with deploying your test application.
+Use the [Optimism Superchain Faucet](https://app.optimism.io/faucet) to get some free ETH on OP Sepolia for multiple OP Chains with just one click.
+You can request testnet ETH from the Superchain Faucet in one of two ways: **connect a wallet** or **login with Github**.
+
+<Image src="/img/builders/dapp-developers/quick-start/faucet-markup.png" alt="Superchain Faucet sign-up options" width={500} height={500} />
+
+*   If you **connect a wallet**, you can get up to 1 testnet ETH per network every 24 hours.
```
Comment (sbvegan) [inline pages/builders/dapp-developers/quick-start.mdx:24]: This should say, "connect a wallet with an Optimist NFT"

File: pages/builders/dapp-developers/quick-start.mdx [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+    The contract will be deployed from the address associated with this private key, so make sure it has enough Sepolia ETH (See [Step 1 above](#step-1-get-testnet-eth-from-superchain-faucet)).
+  </Callout>
+
+  {<h3>Deploy Smart Contracts to Selected Networks</h3>}
+
+  This step deploys all smart contracts in `packages/hardhat/contracts` to the selected network(s). Network names are found in `hardhat.config.js`.
+
+  *   Inside `scaffold-op`, run one of the following options:
+
+  <Tabs items={['optionA', 'optionB']}>
```
Comment (sbvegan) [inline pages/builders/dapp-developers/quick-start.mdx:139]: nit: can we change this to 'Option A' and 'Option B'

File: pages/builders/dapp-developers/quick-start.mdx [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+  <Callout type="info">
+    If the deployments are successful, you will see the deployment transaction hash on the terminal.
+  </Callout>
+</Steps>
+
+Congratulations! You now have an OP Mainnet app ready for development, which can also be deployed to more OP Chains! 🎉
+
+## Next Steps
+
+*   Share feedback about this quick start or `scaffold-op` in the [developer forum](https://github.com/ethereum-optimism/developers/discussions/categories/announcements).
```
Comment (sbvegan) [inline pages/builders/dapp-developers/quick-start.mdx:164]: Lets link the feedback section: https://github.com/ethereum-optimism/developers/discussions/categories/feedback

File: theme.config.tsx [hunk 4]
```diff
@@ -47,10 +47,10 @@ const config: DocsThemeConfig = {
     text: '✏️ Edit this page on GitHub'
   },
   banner: {
-    key: '2.2-release',
+    key: '2.3-release',
     text: (
-      <a href="https://docs.optimism.io/builders/notices/ecotone-changes" target="_self" rel="noopener noreferrer" aria-label="Learn about Ecotone Breaking Changes">
-        📌 UPDATE: Get Prepared for Ecotone Breaking Changes for Wallets, Front-End Developers, Chain Operators, and Node Operators. → 📌
+      <a href="https://docs.optimism.io/builders/dapp-developers/quick-start" target="_self" rel="noopener noreferrer" aria-label="Test out Scaffold-OP for building Superchain Apps">
```
Comment (sbvegan) [inline theme.config.tsx:52]: Screenshot 2024-02-27 at 3 29 54 PM

The code block is hard to read

**Review by sbvegan — CHANGES_REQUESTED** [review]

File: pages/builders/dapp-developers/quick-start.mdx (hunk 1 above)
Comment (cpengilly) [reply pages/builders/dapp-developers/quick-start.mdx:24]: done

File: pages/builders/dapp-developers/quick-start.mdx (hunk 2 above)
Comment (cpengilly) [reply pages/builders/dapp-developers/quick-start.mdx:139]: of course!

File: pages/builders/dapp-developers/quick-start.mdx (hunk 3 above)
Comment (cpengilly) [reply pages/builders/dapp-developers/quick-start.mdx:164]: ty!

File: theme.config.tsx (hunk 4 above)
Comment (cpengilly) [reply theme.config.tsx:52]: fixed, i wasnt sure if it was just me staring at docs all day lol

**Review by sbvegan — APPROVED** [review]

**Pull request merged**